src/frontend/js/components/SaleTunnel/GenericPaymentButton/index.tsx

```diff
@@ -235,8 +242,9 @@ export const GenericPaymentButton = ({ buildOrderPayload }: Props) => {
     const checkOrderValidity = async () => {
       if (round >= PAYMENT_SETTINGS.pollLimit) {
         timeoutRef.current = undefined;
-        orderMethods.abort({ id: payment!.order_id, payment_id: paymentId });
-        setState(ComponentStates.ERROR);
+        // @TODO Instead of displaying a raw error message, we could call onSuccess with a "special" argument
```

Comment:
Why not doing it in this PR?

Reply:
Yup indeed, I'll could do that right now !